modules/apps/29-fee/keeper/events.go

```diff
@@ -39,3 +39,35 @@ func EmitIncentivizedPacketEvent(ctx sdk.Context, packetID channeltypes.PacketId
 		),
 	)
 }
+
+// EmitRegisteredPayeeEvent emits an event containing information of a registered payee for a relayer on a particular channel
+func EmitRegisteredPayeeEvent(ctx sdk.Context, relayer, payee, channelID string) {
```

Comment:
Nit: currently the `Emit...` functions we have in the code base use a verb in the present tense, therefore should we rename this to `EmitRegisterPayeeEvent`? I agree with you that event names that signal that something has occurred should have a verb in the past tense, but maybe we can stay consistent for now with the rest of the codebase and refactor the names of all the functions in a separate PR...

Reply:
I actually thought of this when I added these functions here 😅 
The main reason I opted for past tense was because of the existing `EmitIncentivizedPacketEvent` at the top of the file. I've no problem to update these tho, I'll push an update! 👍